src/main/resources/logback-spring.xml

```diff
@@ -39,8 +39,21 @@
         <appender-ref ref="aiAppender"/>
     </logger>
 
+    <logger name="com.microsoft.applicationinsights" additivity="false" level="INFO">
+        <appender-ref ref="consoleAppender"/>
+        <appender-ref ref="aiAppender"/>
+    </logger>
+
     <root level="INFO">
         <appender-ref ref="consoleAppender"/>
         <appender-ref ref="aiAppender"/>
     </root>
+
+    <springProfile name="ai">
```

Comment:
Why is this necessary?  We don't have a spring profile of `ai` do we?

Reply:
We don't. And its not. Another part of eliminating the differences between this and elite2 around the app insights config. Elite2 does have this section though, though have not used or seen an "ai" spring profile there either.  I've removed it.